src/views/signup/SurveyView1.vue

```diff
@@ -1,18 +1,178 @@
 <template>
-  <div class="flex justify-center">
-    <div class="bg-ivory flex flex-col gap-7 w-full px-4">
+  <div class="flex justify-center w-full">
+    <div class="bg-ivory flex flex-col gap-6 w-full px-6">
       <!-- 타이틀 -->
-      <div class="gap-9">
-        <div class="flex font-bold text-2xl justify-center">사전 조사</div>
-
+      <div class="flex flex-col text-center gap-2">
+        <div class="tfont-bold text-2xl justify-center">사전 조사</div>
         <div class="text-limegreen-700 text-sm">
           현재 자산 유형 분석을 위한 질문입니다.
         </div>
       </div>
-      <!--설문조사 문항-->
-      <div></div>
+
+      <!-- 설문조사 문항 -->
+      <div class="flex flex-col gap-5">
+        <div
+          v-for="question in questions"
+          :key="question.id"
+          class="flex flex-col gap-3 bg-ivory rounded-lg"
+        >
+          <!-- 질문 제목 -->
+          <div
+            class="px-3 py-1 rounded-lg flex bg-limegreen-100 justify-between items-center"
+          >
+            <div class="flex items-center">
+              <div class="font-thin text-sm text-limegreen-800">
+                {{ question.title }}
+              </div>
+              <div
+                v-if="question.subtitle"
+                class="text-xs text-gray-300 font-thin px-1"
+              >
+                {{ question.subtitle }}
+              </div>
+            </div>
+            <div v-if="question.priceUnit" class="text-xs text-gray-300">
+              {{ question.priceUnit }}
+            </div>
+          </div>
+
+          <div class="relative">
+            <div
+              class="absolute top-3 left-6 right-6 h-0.5 bg-limegreen-500 z-0"
+            ></div>
+
+            <!-- 설문조사 응답 버튼과 라벨 -->
+            <div class="grid grid-cols-5 gap-0 relative z-10">
+              <div v-for="(option, index) in question.options" :key="index">
```

Comment:
사소한 것이라 수정요청은 아닙니다!
여기서 key값을 question 객체의 options의 value 속성으로 해도 좋을 것 같습니다!
```suggestion
              <div v-for="(option, index) in question.options" :key="option.value">
```